src/autogluon/bench/datasets/multimodal_dataset.py

```diff
@@ -606,11 +593,9 @@ class JCPennyCategory(BaseMultiModalDataset):
     _INFO = {
         "train": {
             "url": get_repo_url() + "jc_penney_products/train.csv",
```

Comment:
Why did we need SHA's earlier?
and why do we remove it?

Reply:
SHA is not a required field and was in place for sanity check. Since we might have to update datasets sometimes, it is required to update the sha as well which is too much hassle.